xmodule/js/src/tabs/tabs-aggregator.js

```diff
@@ -63,8 +63,8 @@
         if ($.isFunction(onSwitchFunction)) {
           onSwitchFunction();
         }
-        this.$tabs.removeClass('current');
-        $currentTarget.addClass('current');
+        this.$tabs.attr('aria-current', 'false'.removeClass('current'));
```

Comment:
Is this `this.$tabs.attr('aria-current', 'false'.removeClass('current'));` suppose to be `this.$tabs.attr('aria-current', 'false').removeClass('current');`?

Reply:
Thank you for catching this! I have been going crazy trying to find the bug causing the tests to fail.